src/calendar/header/index.js

```diff
@@ -94,7 +94,7 @@ class CalendarHeader extends Component {
       leftArrow = (
         <TouchableOpacity
           onPress={this.onPressLeft}
-          style={this.style.arrow}
+          style={{...this.style.arrow, ...this.style.leftArrow}}
```

Comment:
Why are you spreading the style objects?

Reply:
Because we have two objects. So, we need to create one object for styles. Am i doing that wrong?

Reply:
You don't need to spread the objects.

Reply:
Uh, why? Can you give me example, please.

Reply:
Passing styles as an array: `style={[this.style.arrow, this.style.leftArrow]}`